python/tvm/hybrid/calls.py

```diff
@@ -0,0 +1,92 @@
+"""Intrinsics of TVM-Python Hybrid Script for Python compilation time
+semantic support."""
```

Comment:
the title of this patch, inter function is not accurate, actually it's TVM internal functions or TVM intrinsics

Reply:
I insist mine.

Reply:
Ok, first glance I thought this is supporting calling other python functions defined by users, after reviewing this PR, it's supporting calling TVM functions.